tests/app/phase-tester-response/issue-responded/issue-responded.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { IssueRespondedComponent } from '../../../../src/app/phase-tester-response/issue-responded/issue-responded.component';
+import { Issue, STATUS } from '../../../../src/app/core/models/issue.model';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../constants/githubissue.constants';
+import { IssueComment } from '../../../../src/app/core/models/comment.model';
+import { USER_JUNWEI } from '../../../constants/data.constants';
+
+describe('IssuesPendingComponent', () => {
+  describe('.ngOnInit()', () => {
+    const issuesRespondedComponent = new IssueRespondedComponent();
+    issuesRespondedComponent.ngOnInit();
+    const DUMMY_TEAM = USER_JUNWEI.team;
+    const DUMMY_COMMENT: IssueComment = {
+      id: 1,
+      description: 'This is some description of an issue comment',
+    };
+    let dummyIssue: Issue;
+
+    beforeEach(() => {
+      // generate a well-formed dummy-issue
+      // dummy issues does not have status and comment attributes
+      dummyIssue = Issue.createPhaseTeamResponseIssue(
+        ISSUE_WITH_EMPTY_DESCRIPTION,
+        DUMMY_TEAM
+      );
+    });
+
+    it('should set filter to return true for an issue with comment and is done', () => {
+      // Issue with done status and a comment
+      dummyIssue.status = STATUS.Done;
+      dummyIssue.issueComment = DUMMY_COMMENT;
+      expect(issuesRespondedComponent.filter(dummyIssue)).toBeTrue();
+    });
+
+    it('should set filter to return false for issues that are not done and with comments', () => {
+      // Issue with no status but with comment
+      dummyIssue.issueComment = DUMMY_COMMENT;
+      expect(issuesRespondedComponent.filter(dummyIssue)).toBeFalse();
+
+      // Issue with incomplete status and comment
+      dummyIssue.status = STATUS.Incomplete;
+      expect(issuesRespondedComponent.filter(dummyIssue)).toBeFalse();
+    });
+
+    it('should set filter to return false for issues that are done without comments', () => {
```

Comment:
```suggestion
    it('should set filter to return false for an issue that is marked done, and does not have a comment', () => {
```

`done without comments` is an ambiguous phrase (implies that the issue was updated in a specific manner)
We're simply checking two boolean flags which are independent.